### PR TITLE
Feature/edge video position cx 2404

### PR DESCRIPTION
**Pull request by stefaniacardenas**
## Problem

Edge does not support `object-fit: cover` on Edge. The video width is way larger than its parent container and it needs to be positioned at the center of it. 

## Solution
Use css `transform` and `left` to place the video at the center of its parent.

**Bonus**  This PR also fixes the camera overflow issue on Edge. See screenshots below.

## Before 
screen shot 2018-08-17 at 11 08 16

## After 
screen shot 2018-08-17 at 11 07 21

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [n/a] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [n/a] Have new automated tests been implemented?
- [n/a] Have new manual tests been written down?
- [x] Have tests passed locally?
- [n/a] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-416-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://416-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-416-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-416-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-416-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://416-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Camera/style.css [hunk 1]
```diff
@@ -142,9 +140,8 @@
 @supports (-ms-ime-align:auto) {
   .video {
     width: auto;
-    /* HACK: this will make the video (kind of) look centered on small screens in absence of `object-fit:cover`.
-    It might not work on all screen sizes. This will be handled as part of another story*/
-    left: -50%;
+    left: 50%;
+    transform: translate(-50%, 0%) !important;
```
Comment (stephencookdev) [inline src/components/Camera/style.css]: what's this `!important` needed for?

File: src/components/Theme/style.css [hunk 2]
```diff
@@ -9,6 +9,15 @@
   flex-direction: column;
 }
 
+/* this will only apply styles to Edge */
+@supports (-ms-ime-align:auto) {
+  /* Prevent overflow on fullscreen on Edge */
+  .fullScreenStep {
```
Comment (stephencookdev) [inline src/components/Theme/style.css]: I get that it's not a problem on other browsers - but is there any reason we can't do this for all browsers?

Comment (stefaniacardenas) [reply src/components/Theme/style.css]: No reason. I've changed it :)

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-416-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://416-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-416-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Camera/style.css [hunk 3]
```diff
@@ -125,24 +125,16 @@
...
-    width: auto;
-    left: 50%;
-    transform: translate(-50%, 0%) !important;
-  }
+.webcamContainer {
+  position: absolute;
+  top: 0;
+  left: 50%;
+  margin: auto;
+  transform: translate(-50%, 0%) !important;
```
Comment (rfreitas) [inline src/components/Camera/style.css]: is the `!important` needed?

Comment (stefaniacardenas) [reply src/components/Camera/style.css]: no, sorry this is still WIP. I need to update `react-webcam` to extend the `transform`

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-416-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://416-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-416-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Camera/style.css (hunk 3 above)
Comment (stefaniacardenas) [reply src/components/Camera/style.css]: No, it's not! :)

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-416-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-416-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://416-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-416-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**